src/modules/dashboard/components/ExpenditurePage/Payments/Payments.tsx

```diff
@@ -0,0 +1,159 @@
+});
+
+interface Props {
+  sidebarRef: HTMLElement | null;
+}
+
+const Payments = ({ sidebarRef }: Props) => {
+  const [, { value: recipients }, { setValue }] = useField('recipients');
+  const { colonyName } = useParams<{
+    colonyName: string;
+  }>();
+
+  const { data: colonyData } = useColonyFromNameQuery({
+    variables: { address: '', name: colonyName },
+  });
+  const { colonyAddress } = colonyData || {};
+
+  const { data: colonyMembers, loading } = useMembersSubscription({
+    variables: { colonyAddress: colonyAddress || '' },
+  });
+
+  const onToogleButtonClick = useCallback(
+    (index) => {
+      setValue(
+        recipients.map((recipient, idx) =>
+          index === idx
+            ? { ...recipient, isExpanded: !recipient.isExpanded }
+            : recipient,
+        ),
+      );
+    },
+    [recipients, setValue],
+  );
+
+  return (
+    <div className={styles.paymentContainer}>
+      <div className={styles.recipientContainer}>
+        <div className={styles.payments}>
+          <FormattedMessage {...MSG.payments} />
+        </div>
+        {loading ? (
+          <SpinnerLoader appearance={{ size: 'medium' }} />
+        ) : (
+          <FieldArray
+            name="recipients"
+            render={({ push, remove }) => (
+              <>
+                {recipients.map((recipient, index) => (
+                  <div className={styles.singleRecipient} key={recipient.id}>
+                    <FormSection>
+                      <div className={styles.recipientLabel}>
+                        {recipient.isExpanded ? (
+                          <>
+                            <Icon
+                              name="collapse"
+                              onClick={() => onToogleButtonClick(index)}
+                              className={styles.signWrapper}
+                              title={MSG.minusIconTitle}
+                            />
+                            <div
+                              className={classNames(styles.verticalDivider, {
+                                [styles.dividerInLastItem]:
+                                  index === recipients?.length - 1,
+                              })}
+                            />
+                          </>
+                        ) : (
+                          <Icon
+                            name="expand"
+                            onClick={() => onToogleButtonClick(index)}
+                            className={styles.signWrapper}
+                            title={MSG.plusIconTitle}
+                          />
+                        )}
+                        {index + 1}: <FormattedMessage {...MSG.recipient} />
+                        {recipients.length > 1 && (
+                          <Icon
+                            name="trash"
+                            className={styles.deleteIcon}
+                            onClick={() => remove(index)}
+                            title="Delete recipient"
```

Comment:
It's better to use message descriptor as title, like you have for the expand icon. Please, update.